components/budget/ExpenseBudgetItem.js

```diff
@@ -279,18 +291,20 @@ const ExpenseBudgetItem = ({
             />
           )}
         </Box>
-        {showProcessActions && expense?.permissions && (
-          <ButtonsContainer>
-            <ProcessExpenseButtons
-              host={host}
-              collective={expense.account || collective}
-              expense={expense}
-              permissions={expense.permissions}
-              buttonProps={{ ...DEFAULT_PROCESS_EXPENSE_BTN_PROPS, mx: 1, py: 2 }}
-              onSuccess={onProcess}
-            />
-          </ButtonsContainer>
-        )}
+        {showProcessActions &&
+          expense?.permissions &&
```

Comment:
This is not enough, logged out users will see this instead of the normal tag:

![image](https://user-images.githubusercontent.com/1556356/131301529-0a2972b5-9c8d-423b-91c2-c68730284253.png)

I suggest using the `hasProcessButtons` helper. We may need to add `canMarkAsSpam` in there.